js/tiles.js

```diff
@@ -1104,4 +1104,33 @@ const desert = {
             this.ground.show();
         }
     }),
+    lava: Object.assign({}, OpenDoors, {
+        canEnterFromTheRight() {
+            return false;
+        },
+        canEnterFromTheBottom() {
+            return true;
+        },
+        canLeaveToTheBottom() {
+            return true;
+        },
+        canLeaveToTheRight() {
+            return false;
+        },
+        visit: function() {
+            swal({
```

Comment:
We have an API for that because the dialog functionality might change in the future.

Reply:
see readme.md